pkg/virtctl/pause/pause.go

```diff
@@ -114,3 +107,14 @@ func executePauseCMD(client kubecli.KubevirtClient, namespace, resourceType, res
 
 	return nil
 }
+
+func handleNotFoundError(vm *kubevirtV1.VirtualMachine, err error) error {
```

Comment:
`err` variable is always shadowed with the returned value of `vm.RunStrategy()`.
You can drop it:
```golang
func handleNotFoundError(vm *kubevirtV1.VirtualMachine) error {
```